src/styles/blocks/header-nav.scss

```diff
@@ -0,0 +1,58 @@
+.nav {
+  &__list {
+    display: flex;
+    text-align: center;
+    list-style: none;
+    text-decoration: none;
```

Comment:
The `text-decoration: none;` property is not necessary for list items (`<li>`), as it is typically used for links (`<a>`). Consider removing it from this line.